app/services/open_library.rb

```diff
@@ -0,0 +1,27 @@
+class OpenLibrary
+  include HTTParty
+  base_uri 'https://openlibrary.org'
+
+  def initialize(isbn)
+    @isbn = "ISBN:#{isbn}"
+    @options = { query: { bibkeys: "ISBN:#{isbn}", format: 'json', jscmd: 'data' } }
+  end
+
+  def books
+    response = self.class.get('/api/books', @options)
+    raise 'External error' if response.code != 200
```

Comment:
I thinks you may do a 
```ruby
raise 'External error' if response.ok?
```
Instance of that... but it's fine.

Reply:
i think a best way

```suggestion
    raise 'External error' unless response.ok?
```

Reply:
Yes, that is it! :+1: :+1: